tests/test_authz_authorizer.py

```diff
@@ -39,10 +39,8 @@ def test__init__(self):
         assert self.authz.permission == "permission_name"
 
     def test__repr__(self):
-        assert (
-            repr(self.authz)
-            == "Authorizer(auth_jwt=<jwt>, resource_name='test_resource', permission_name='permission_name')"
-        )
+        expected_repr = "Authorizer(auth_jwt=<jwt>, resource_name='test_resource', permission_name='permission_name')"  # noqa: E501
```

Comment:
same as above

Reply:
It looks like you can eliminate every `noqa: E501` once you disabled it globally 😉